### DISCUSSION TITLE
Personalized LEO/EMS/Dispatch Dashboard

### DISCUSSION THREAD

**Discussion by casperiv0** [SnailyCAD/snaily-cadv4#918] [Polls]
Something that I've been thinking of for some time is that a user could setup their dashboard the way they'd like by adding the ability to move different sections on the dashboard. I'd like to get some feedback for this before I implement it since it's quite complicated:).

A simple design idea here: https://www.figma.com/file/HhQz0Uf242ac07JdymASfT/Untitled?node-id=2%3A3, feedback is welcome, **Note:** this is not the final design.

**Comment by jxckUK** [comment]
Yes - more customisation can only ever be a good thing.

**Comment by subtosharki** [comment]
yes, it would make users feel more at home if they could make their screen how they liked

**Comment by HearingSubset19** [comment]
Yeah, Everyone has different aspects to them and people might benefit over it.

**Comment by casperiv0** [comment]
A simple design idea here: https://www.figma.com/file/HhQz0Uf242ac07JdymASfT/Untitled?node-id=2%3A3, feedback is welcome, **Note:** this is not the final design.
